tests/test_hyperbolic_coords.py

```diff
@@ -70,21 +70,21 @@ def test_extrinsic_half_plane_extrinsic(self):
 
         x2 = Hyperbolic.change_coordinates_system(
             x_up, 'half-space', 'extrinsic')
-        self.assertAllClose(x, x2)
+        self.assertAllClose(x, x2, gs.atol)
```

Comment:
do we need this `gs.atol` here? Isn't it used by default in `assertAllClose`?

Reply:
you're right, it's an oversight !